src/extensions/default/InlineColorEditor/main.js

```diff
@@ -27,8 +27,26 @@ define(function (require, exports, module) {
     var EditorManager       = brackets.getModule("editor/EditorManager"),
         ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
         InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
-        ColorUtils          = brackets.getModule("utils/ColorUtils");
+        ColorUtils          = brackets.getModule("utils/ColorUtils"),
+        properties          = JSON.parse(require("text!ColorProperties.json"));
 
+    var DEFAULT_COLOR = "white";
+
+    /*
+    This function takes a line number, string and the colon position, and writes
+    the default color value at 2 places after the colon position
+     */
+    function setLine(lineNo, newContent, hostEditor, colonPos) {
```

Comment:
I think it's odd that we're resetting the entire contents of the editor for a single line change when we can just use `replaceRange` instead to add in what we need.